extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -134,6 +134,8 @@ export const server = localize('server', "Server");
 export const defaultUser = localize('default', "default");
 export const selectProfileToUse = localize('selectProfileToUse', "Select publish profile to load");
 export const selectProfile = localize('selectProfile', "Select Profile");
+export const saveProfileAsButtonText = localize('saveProfileAsButtonText', "Save Profile As...");
+export const save = localize('save', "Save");
```

Comment:
I don't see `save` being used anywhere. Can this be removed?

Reply:
It is used on the save dialog.